tools/certa/src/runners/chrome/ChromeTestRunner.ts

```diff
@@ -18,38 +18,51 @@ interface ChromeTestResults {
 
 type ConsoleMethodName = "log" | "error" | "dir";
 
-let browser: puppeteer.Browser;
+let browser: Browser;
 let webserverProcess: ChildProcess;
 
 export class ChromeTestRunner {
   public static readonly supportsCoverage = true;
   public static readonly supportsCleanup = true;
   public static async initialize(config: CertaConfig): Promise<void> {
-    // Go ahead and launch puppeteer now - the VS Code debugger gets confused if it can't at least see the chrome instance right away.
-    const options = {
-      ignoreHTTPSErrors: true,
+    // Go ahead and launch playwright now - the VS Code debugger gets confused if it can't at least see the chrome instance right away.
+    const options: LaunchOptions = {
       args: config.chromeOptions.args,
       headless: !config.debug,
     };
 
     if (config.debug)
-      options.args.push(`--disable-gpu`, `--remote-debugging-port=${config.ports.frontendDebugging}`);
+      options.args?.push(
```

Comment:
do you have prettier or some other formatter turned on?

Reply:
i usually end up manually commenting it out in the workspace settings while i work in the core repo